### PR TITLE
DCC-5432. ES upgrade

### PR DESCRIPTION
**Pull request by slovit**

### CODE REVIEW

File: dcc-repository-client/pom.xml [hunk 1]
```diff
@@ -110,6 +110,16 @@ ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF S
...
     </dependency>
+        <dependency>
+    	<groupId>org.apache.logging.log4j</groupId>
+    	<artifactId>log4j-api</artifactId>
+    	<version>2.6.2</version>
+    </dependency>
+        <dependency>
+    	<groupId>org.apache.logging.log4j</groupId>
+    	<artifactId>log4j-core</artifactId>
+    	<version>2.6.2</version>
```
Comment (btiernay) [inline dcc-repository-client/pom.xml]: Use variable instead, thanks!

**Comment by andricDu** [comment]
test this please

**Comment by andricDu** [comment]
ok to test

**Comment by andricDu** [comment]
retest this please

**Comment by andricDu** [comment]
👍

**Pull request merged**